### PR TITLE
Update for the latest version 0.17.0

### PR DESCRIPTION
**Pull request by bleiva**
Update README for the latest version 0.17.0

### CODE REVIEW

File: docs/networks.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ indexer. The only requirement is a minimum stake of 100k GRT.
 | `INDEXER_AGENT_NETWORK_SUBGRAPH_ENDPOINT`   | `--network-subgraph-endpoint`   | `https://gateway.thegraph.com/network`                  |
 | `INDEXER_AGENT_DAI_CONTRACT`                | `--dai-contract`                | `0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48` (USDC)     |
 | `INDEXER_AGENT_COLLECT_RECEIPTS_ENDPOINT`   | `--collect-receipts-endpoint`   | `https://gateway.network.thegraph.com/collect-receipts` |
+| `INDEXER_AGENT_GAS_COST_MAX`                | `--gas-cost-max`                | `50`                                                    |
```
Comment (fordN) [inline docs/networks.md:61]: `--gas-price-max`?

**Review by fordN — CHANGES_REQUESTED** [review]
Thanks for submitting this @bleiva! 🙌 

One small change requested.

File: docs/networks.md (hunk 1 above)
Comment (bleiva) [reply docs/networks.md]: Yeah, you are right @fordN. I've updated it

**Review by fordN — APPROVED** [review]
👍

**Pull request merged**